ERP/resources/views/components/navbar.blade.php

```diff
@@ -13,6 +13,14 @@
                 <li class="nav-item">
                     <a class="nav-link active" href="{{route('user.management')}}">User Management</a>
                 </li>
+                <li class="nav-item">
+                    <a class="nav-link active" href="{{route('inventory')}}">Inventory</a>
+                </li>
```

Comment:
Here it seems like the Inventory navbar entry also shows up also for the IT user?

Reply:
fixed